run

```diff
@@ -155,12 +155,45 @@ while [[ -n "${1:-}" ]] && [[ "${1:0:1}" == "-" ]]; do
             shift
         ;;
         -h|--help) echo "$USAGE"; exit ;;
-        -v|--version) echo "Generated from generator-canonical-webteam@3.2.1"; exit ;;
+        -v|--version) echo "Generated from generator-canonical-webteam@3.3.1"; exit ;;
```

Comment:
Sorry, this was actually a bug in the generator (now fixed, for whenever the next version is released) - but could you please change this to `3.3.3`?

```suggestion
        -v|--version) echo "Generated from generator-canonical-webteam@3.3.3"; exit ;;
```